docs/api/base-window.md

```diff
@@ -511,6 +511,10 @@ A `string` property that defines an alternative title provided only to
 accessibility tools such as screen readers. This string is not directly
 visible to users.
 
+#### `win.snapped` _Windows_
```

Comment:
Do we have any way of marking properties as readonly in our docs?

Reply:
@itsananderson yes! good callout, done in d23d8cbe33